docker-compose.yml

```diff
@@ -155,13 +155,14 @@ services:
     labels:
       - traefik.enable=true
       - traefik.http.services.cvat-ui.loadbalancer.server.port=80
-      - traefik.http.routers.cvat-ui.rule=Host(`${CVAT_HOST:-localhost}`)
+      - traefik.http.routers.cvat-ui.rule=Host(`${CVAT_HOST:-localhost}`) &&
+        ! PathPrefix(`/api/`, `/git/`, `/opencv/`, `/static/`, `/admin`, `/documentation/`, `/django-rq`)
```

Comment:
@azhavoro , why do we need the line now? How is it connected with the patch?

Reply:
@nmanovic I think it's better to remove the `HEALTHCHECK` command from the Dockerfile for this PR, this currently leads to routing issues related to Traefik container health check behavior https://github.com/traefik/traefik/issues/7732. At the moment I don't find a suitable workaround, so I propose to create a separate issue about adding `HEALTHCHECK` instruction to CVAT server docker image.

Reply:
OK. Let's do that in this case.